docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -17,6 +17,28 @@ user with access to the <<cluster-update-settings,cluster-update-settings>>
 API can make the cluster read-write again.
 
 
+[[cluster-max-tombstones]]
+==== Index Tombstones
+
+The cluster state maintains index tombstones to explicitely denote indices that 
+have been deleted.  The tombstones are maintained to prevent indices that were 
+deleted when a node was offline from being reimported as dangling indices when 
+the node comes back online.  The number of tombstones maintained in the cluster 
+state is controlled by the following property:
+
+`cluster.indices.tombstones.size`::
+
+The maximum number of tombstones that can be in the cluster state before tombstones 
+are purged.  The current default is 500.  Index tombstones remain in the cluster state 
+forever, until the maximum is exceeded, at which point tombstones are purged to stay 
+within the maximum allowed.  If your use case involves a large amount of regular index 
```

Comment:
Why not something like "index tombstones prevent nodes that are not part of the cluster when a delete occurs from joining the cluster and reimporting the index as though the delete was never issued. To keep the cluster state from growing huge we only keep the last `cluster.indices.tombstones.size` deletes. It defaults to `500` but you can increase it if you expect nodes to be absent from the cluster and miss more than `500` deletes. We think that is rare, thus the default. We also think that these don't take up a ton of space in the cluster state but that `50,000` is probably too big.